README.md

```diff
@@ -6,136 +6,160 @@
 
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-This library offers a simple contract to use the biometry across Android, iOS and UWP.
+This library offers a simple contract to use the biometry across Android, iOS and Windows (UWP & WinUI).
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 
 ## Features
 
-The biometryService Interface 'IBiometryService' Implement the following method :
+The Biometry Service Interface `IBiometryService` is made of the following methods:
 
 - GetGapabilites
-- ValidateIdentity
+- ScanBiometry
 - Encryt
 - Decrypt
+- Remove
 
 As of now, this is the list of features available per platform.
 
-
-| Feature          | iOS | Android | UWP  |
-| ---------------- | --- | ------- | ---- |
-| GetCapability    | x   | x       | x    |
-| ValidateIdentity | x   | x       | Mock |
-| Encrypt          | x   | x       | Mock |
-| Decrypt          | x   | x       | Mock |
+| Feature          | iOS     | Android | UWP     | WinUI   |
+| ---------------- | ------- | ------- | ------- | ------- |
+| GetCapability    | &check; | &check; | &check; | &check; |
+| ValidateIdentity | &check; | &check; | &cross; | &cross; |
+| Encrypt          | &check; | &check; | &cross; | &cross; |
+| Decrypt          | &check; | &check; | &cross; | &cross; |
 
 ## Getting Started
 
-Install the latest version of BiometryService Nuget package "Add Nuget Reference".
+Install the latest stable version of `BiometryService` in your platform heads and `BiometryService.Abstractions` in your presentation layer if you are using MVVM pattern, and if not just install both in your platform heads.
 
-A small sample is available as a playground.
+A small sample application is available as a playground.
```

Comment:
Should we specify a bit more where it it? Like saying there's an uno project inside this repo. May be overkill, but if we want to have an open source perspective, maybe it should be explicitly mentionned